panel/tests/chat/test_interface.py

```diff
@@ -358,6 +358,10 @@ def post_callback(instance, event):
         assert chat_interface.objects[1].object == "2"
         assert chat_interface.objects[2].object == "3"
 
+    def test_manual_user(self):
+        chat_interface = ChatInterface(user="New User")
+        assert chat_interface.user == "New User"
```

Comment:
I'd recommend also testing the message object
```suggestion
        assert chat_interface.user == "New User"
        chat_interface.send("Test")
        assert chat_interface.objects[0].user == "New User"
```